apps/omg_watcher_info/test/omg_watcher_info/db/txoutput_test.exs

```diff
@@ -43,4 +43,19 @@ defmodule OMG.WatcherInfo.DB.TxOutputTest do
     assert not is_nil(utxo)
     assert utxo.amount == big_amount
   end
+
+  @tag fixtures: [:initial_blocks]
+  test "spend_utxos sets timestamps correctly" do
```

Comment:
I think it should be possible to follow the same way as the test above: `DB.Block.insert_with_transactions/1` followed by `utxo = DB.TxOutput.get_by_position/1` to get the utxo.

Then pass that utxo into `DB.TxOutput.spend_utxos/1` then assert for the changed dates.

Reply:
2nd thought if we're moving away from fixtures, then we can just directly pre-insert some utxos and use them in `spend_utxos/1`.

Reply:
yes!!! if tests can be self contained, that would make test simple and readable! thats what we want and should strive for.

Reply:
Totally agree. Will update the PR and remove fixtures from tests.